crates/wash-lib/src/cli/start.rs

```diff
@@ -79,24 +79,27 @@ pub async fn handle_start_actor(cmd: StartActorCommand) -> Result<CommandOutput>
         .into_ctl_client(Some(cmd.auction_timeout_ms))
         .await?;
 
+    let actor_ref = if cmd.actor_ref.starts_with('/') {
+        format!("file://{}", &cmd.actor_ref) // prefix with file:// if it's an absolute path
+    } else {
+        cmd.actor_ref.to_string()
+    };
```

Comment:
Just a note for something I considered for other reviewers, this is a better check than checking to see if the ref is an absolute path on the system as wash could be starting the actor from a path on a different machine 👍🏻